### PR TITLE
Tabs

### PR DESCRIPTION
**Pull request by Sonicrida**
**UI 95 Tabs**

This takes in an array of components and displays them evenly spaces for mobile navigation. I wasn't sure if Should I match the styles 100% to the design or would the styles make more sense to be handled at library implementation (ex: styling the components being passed in)?

image
- Example is an array of two buttons and one link.

### CODE REVIEW

**Comment by nickmccurdy** [comment]
Good question. I originally thought we would copy MyCrypto's UI since we only have one design for it so far, but we could alternatively require the user to style the bar and tags separately and only implement the logical parts.

File: src/atoms/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          <Tab>
+            <Typography>{child}</Typography>
+          </Tab>
+        ))}
+      </TabsContainer>
+    );
+  }
+}
+
+export default Tabs;
```
Comment (nickmccurdy) [inline src/atoms/Tabs/Tabs.tsx]: I think we need a way to control the current tab, think of it more like a select menu or toggle

**Comment by Sonicrida** [comment]
Just pushed up new styles to match the original designs when passing in links:

image

**Comment by nickmccurdy** [comment]
Ready to merge if you'd like

**Comment by Sonicrida** [comment]
That makes a lot of sense! I'll go ahead and merge this.

**Pull request merged**